src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -1334,6 +1344,8 @@ subroutine initialize_dyn_split_RK2(u, v, h, uh, vh, eta, Time, G, GV, US, param
   CS%id_veffA = register_diag_field('ocean_model', 'veffA', diag%axesCvL, Time, &
        'Effective V-Face Area', 'm^2', conversion = GV%H_to_m*US%L_to_m, &
        x_cell_method='sum', v_extensive = .true.)
+  CS%id_deta_dt = register_diag_field('ocean_model', 'deta_dt', diag%axesT1, Time, &
+      'Barotropic SSH tendency', trim(thickness_units)//' s-1', conversion=GV%H_to_MKS/US%T_to_s)
```

Comment:
Other diagnostics in the code use `*US%s_to_T` in the conversion argument for rate quantities, e.g., `id_dh_dt`, `id_dynamics_h_tendency`, instead of `/US%T_to_s`. I think it would be helpful to follow that convention here, to avoid the future reader's question "Why is it done differently here?"

Reply:
Good catch @klindsay28